src/textfield.vue

```diff
@@ -92,6 +94,16 @@ export default {
   methods: {
     fireInputEvent: function (event) {
       this.$emit('input', event.target.value)
+    },
+    focus () {
+      this.$refs.focusTarget.focus()
+    },
+    focusAndSelect () {
```

Comment:
Sorry - misunderstood what you meant :). Looks like it's all good now though. Tx.